src/js/components/Loader.js

```diff
@@ -52,7 +52,9 @@ class Loader extends React.Component {
   render() {
     const { className, innerClassName, flip, size, type } = this.props;
     const config = typeMap[type] || typeMap.ballScale;
-    const classes = classNames("loader horizontal-center", className);
+    const classes = this.props.supressHorizontalCenter
```

Comment:
@weblancaster @tnguyen0 @bstavroulakis let's make sure we add a new prop to the propTypes, including a default. I think a BOOL would be sensible here.